src/types/pull-requests.ts

```diff
@@ -0,0 +1 @@
+export type CodeReviewStatus = "APPROVE" | "REQUEST_CHANGES" | "COMMENT";
```

Comment:
Is it really necessary that it needs its own file

Reply:
If by the time I'm done it's still solo I'll group it but I expect to fill more types here